database/seeders/EventsErstiwocheSeeder.php

```diff
@@ -345,14 +338,14 @@ public function runSport(): void
         $event->description = '<p>Auch sportliche Aktivitäten kommen bei uns nicht zu kurz. Für eine Anmeldegebühr von <strong>5€</strong> könnt ihr euch am Freitag auspowern.</p>
             <p>Bitte bezahlt die Anmeldegebühr am Montag oder Mittwoch im FSR. Solltet ihr bis Mittwoch nicht gezahlt haben, werden eure reservierten Plätze wieder freigegeben.</p>
             <p>Bitte beachtet auch die folgenden Hinweise zu den einzelnen Programmpunkten:</p>
-            <p><strong>Fußball, Volleyball, Basketball:</strong> Die Anmeldegebühr bekommt ihr zurückerstattet, wenn es wegen schlechtem Wetter ausfällt.</p>
+            <p><strong>Hochschulsport:</strong>Bitte bringt 5€ an Pfand mit</p>
```

Comment:
Please use `Fußball, Volleyball, Basketball` as name and not `Hochschulsport`, this is easier to understand for our first semester students.